src/docs/content/observability/observability-example/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Observability Example"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+weight: 110
+showChildren: true
+---
+
+## Introduction
+
+In this section we will try to understand Observability on Oracle Cloud Infrastructure (OCI) by simulating a failure scenario on MuShop application.
+
+![observability-scenario](../images/observability-scenario.png)
+
+## Scenario Details
+- User logs in to MuShop and Orders items with value more than $105.
+- User notices the Order processing failure.
+- User gets notified on the Payment failures. 
+- User analyzes the failure using Oracle Cloud Infrastructure(OCI) Observability Services.
+
+## Following topics will be covered
```

Comment:
Since the contents are now limited to just `setup` and `usecase`, I think it's better to simply say "Next" here